homeassistant/components/home_connect/sensor.py

```diff
@@ -81,6 +83,18 @@ class HomeConnectSensorEntityDescription(SensorEntityDescription):
         desc="Drip Tray Full",
         appliance_types=("CoffeeMaker",),
     ),
+    HomeConnectSensorEntityDescription(
+        key="Salt Nearly Empty",
+        translation_key="alarm_sensor_salt_nearly_empty",
+        state_key=DISHWASHER_EVENT_SALT_NEALY_EMPTY,
```

Comment:
```suggestion
        state_key=DISHWASHER_EVENT_SALT_NEARLY_EMPTY,
```

Reply:
Actually, you wouldn't use the `state_key` since it's not used in the `HomeConnectAlarmSensor` code and also not defined in the entity description class. You can set key to this constant and also remove the translation key, unless you have a `strings.json` that you'll be updating. Also, `desc` would have the `key` parameter value.